py/desitarget/cuts.py

```diff
@@ -283,7 +388,7 @@ def unextinct_fluxes(objects):
     else:
         return result
 
-def apply_cuts(objects):
+def apply_cuts(objects,QSOSelection):
```

Comment:
Adding `QSOSelection` without a default has broken the unit tests.  Whenever contributing new code, please always run "python setup.py test" from the top-level directory to make sure that tests pass before pushing them to github.

Please add a description of the valid `QSOSelection` values to the docstring (the triple-quoted string just after the function definition).  From a quick look, I think you implemented this as an integer.  Let's use a more user-friendly string, e.g. QSOSelection='colorcuts' or QSOSelection='bdt' .

Let's also add a default for `QSOSelection` to `apply_cuts` so that `apply_cuts(objects)` will still work, using whatever the current default choice should be.  e.g.
```python
def apply_cuts(objects, QSOSelection='colorcuts')
```
That would fix the failing unit tests, though another unit test should also be added to explicitly test the BDT method too.

I would prefer "qso_selection" or something in lower case to match our python style.  Generally starting with a Capital letter is reserved for classes, not variables.